app/dataflow/reducers/index.js

```diff
@@ -1,43 +1,44 @@
-import { routerReducer as routing } from 'react-router-redux';
-import { combineReducers } from 'redux';
-import * as types from '../actions/types';
-import fetchedForecast from '../../utils/testForecast';
-import fetchedWeather from '../../utils/testWeather';
-import fetchedLocation from '../../utils/testLocation';
+import { routerReducer } from 'react-router-redux';
+import reduceReducers from 'reduce-reducers';
+import weatherReducer from './weather';
+import mainReducer from './main';
+import locationReducer from './location';
+import { load } from '../../utils/localStorage';
+import { DEFAULT_METRIC } from '../../utils/constants';
+
+const stored = load('store') || {};
+const { city, countryCode, metric, main } = stored;
 
 const initialState = {
-    geolocation: fetchedLocation,
-    city: 'Minsk',
-    weather: fetchedWeather,
-    forecast: fetchedForecast,
+    geolocation: {
+        code: -1,
+        message: 'You location is not defined',
+    },
+    weather: {
+        status: 0,
+        message: 'No location API available to get initial state',
+        weatherTypes: [{ main }],
+    },
+    forecast: [{
+        status: 0,
+        message: 'No location API available to get initial state',
+    }],
+    forecastFilter: '12H',
+    nearestCities: [],
+    countryCode,
+    city,
+    metric: metric || DEFAULT_METRIC,
```

Comment:
@AlexeyBarkow move metric a bit higher, shorthade object properties should be grouped at the end or beginning